transactions/lockedTokens/admin/recover_lease_tokens.cdc

```diff
@@ -0,0 +1,40 @@
+import "FungibleToken"
+import "FlowToken"
+import "LockedTokens"
+
+transaction(amount: UFix64, to: Address) {
+
+    // The Vault resource that holds the tokens that are being transferred
+    let sentVault: @{FungibleToken.Vault}
+
+    prepare(signer: auth(BorrowValue) &Account) {
+
+        // Get a reference to the signer's locked token manager
+        let tokenManagerRef = signer.storage.borrow<auth(FungibleToken.Withdraw, LockedTokens.RecoverLease) &LockedTokens.LockedTokenManager>(from: LockedTokens.LockedTokenManagerStoragePath)
+            ?? panic("The signer does not store a LockedTokenManager object at the path "
+                    .concat(LockedTokens.LockedTokenManagerStoragePath.toString()))
+
+        let nodeRef = tokenManagerRef.borrowNodeForLease()
+            ?? panic("Could not borrow a reference to a node in the LockedTokenManager of the signer's account")
+
+        // Withdraw enough tokens to pay for fees, assuming there are some rewards in the rewards bucket
+        tokenManagerRef.deposit(from: <-nodeRef.withdrawRewardedTokens(amount: 0.0001)!)
```

Comment:
Is this reimbursing the fee incurred for running this transaction?

Reply:
yeah, most of these accounts have only the minimum in their main vaults so they need to be topped up a little bit to pay for the tx fee